projects/fmcjesdadc1/src/devices/adi_hal/parameters.h

```diff
@@ -64,15 +64,12 @@
 #define GPIO_OFFSET				54
 #endif
 
-#define ADC_0_DDR_BASEADDR			(XPAR_DDR_MEM_BASEADDR + 0x800000)
-#define ADC_1_DDR_BASEADDR			(XPAR_DDR_MEM_BASEADDR + 0x900000)
+#define ADC_DDR_BASEADDR			(XPAR_DDR_MEM_BASEADDR + 0x800000)
```

Comment:
This needs to be modified on line 53 too.
![image](https://user-images.githubusercontent.com/16192750/128351391-7c9bb5f3-00b0-4829-b701-12a42105a03d.png)

Reply:
Also a change in app_iio.c is needed too:
![image](https://user-images.githubusercontent.com/16192750/128351548-1209aefd-4616-443f-970e-484ddf7ffd3b.png)